src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,28 @@
+package duke.task;
+
+public class Deadline extends Task {
+    private String dueTime;
+
+    public Deadline(String name, String dueTime) {
+        super(name);
+        this.dueTime = dueTime;
+        this.type = 'D';
+    }
+
+    public Deadline(String name, String dueTime, boolean isDone) {
+        super(name, isDone);
+        this.dueTime = dueTime;
+        this.type = 'D';
+    }
+
+    @Override
+    public String toString() {
+        String done = isDone ? "X" : " " ;
+        return "[" + type + "]"+ "["+ done +"] " + name + " (" + dueTime + ")";
```

Comment:
Try to standardise the spacing before and after operators such as "+" in your code.